wallet/src/Pos/Wallet/Web/Tracking/Sync.hs

```diff
@@ -446,6 +448,12 @@ applyModifierToWallet wid newTip CAccModifier{..} = do
     sortedAddedHistory <-
         getNewestFirst <$> sortWalletThByTime wid (DL.toList camAddedHistory)
     WS.updateHistoryCache wid $ sortedAddedHistory <> oldCachedHist
+    let cMetas = map (bimap encodeCType (CTxMeta . timestampToPosix)) $
+                 catMaybes $
+                    zipWith (\a b -> (a,) <$> b)
```

Comment:
Minor: why not just some `map (\THEntry {..} -> _thTxId <$> _thTimestamp) sortedAdddHistory`

Reply:
Yes, it's even better, I just was too lazy to find how this constructor is called 😜

Reply:
Lol, didn't notice your previous comment

Reply:
> Note that sortWalletThByTime uses timestamps in meta, and you initialize meta after calling sortWalletThByTime
will this again break apply rollback @flyingleafe ?

Reply:
We discussed with @pva701 , it doesn't break anything.
However this thing is changed in another PR which is also merged